### DISCUSSION TITLE
error of iTwin / mobile-sdk-samples

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#2827] [Q&A] [answered]
when i open the mobilestarter project by xcode on mac from iTwin / mobile-sdk-samples,and show error
how to solve this problem

![20211202180711](https://user-images.githubusercontent.com/18022720/144574933-0a26aad2-51af-441a-9b22-7d35326a2f27.png)

![QQ图片20211203170426](https://user-images.githubusercontent.com/18022720/144575562-cf293f7f-ad39-42e9-afc0-f42db0d9b941.png)

**Comment by Carl-Hinkle** [comment, marked as answer]
Hi, did you try resetting the package cache as mentioned in the README?

![Screen Shot 2021-12-03 at 7 34 04 AM](https://user-images.githubusercontent.com/47985924/144603428-fa1ccc68-be9c-4d7d-bdd7-a4173f6bf199.png)

The screenshot is from Xcode 13.  It might be a little different in 12.

**Reply by paprophet** [reply]
yes,i do "Reset Package Caches" many time before send the discuss,but the error is exist.

**Reply by Carl-Hinkle** [reply]
I haven't been able to replicate this, but searching on Google suggests that you might be behind a proxy and need to configure Git.  You might also try updating to the latest version of Xcode and open the project again.

**Reply by Carl-Hinkle** [reply]
Also, if you're on an M1 Mac, the simulator build will not work at this time.  You'll need to use a physical device.  The simulator does work on Intel Macs.

**Reply by paprophet** [reply]
i use xcode 13 by a  other mac book,and not show this error, and link the physical device (ipad),
but when i take the imodel file to the app,and the app show black,and not show the imodel, but show the view list from the imodel file.
 

![2](https://user-images.githubusercontent.com/18022720/145162187-9bbe46a2-cd08-439e-a9d2-243162a33bd2.jpg)

**Reply by Carl-Hinkle** [reply]
Can you please start a new discussion about this since it's a different issue?  And please share the iModel with us if possible and provide specifics about the device, i.e. what model of iPad are you using.  I'm going to mark this discussion as resolved since the problem was with your setup.

**Reply by paprophet** [reply]
ok,question of the not show model ,i will start a new discussion .

https://github.com/iTwin/mobile-ios-package/blob/main/Package.swift
 platforms:
[
.ios("12.2")
]

so,i can not git the package by xcode (swift package dependencies) on the low version of mac (mac OS Catalina 10.15.7)?
